### PR TITLE
MAISTRA-2065: Update RPMs for 1.1.11

### PR DESCRIPTION
**Pull request by knrc**

### CODE REVIEW

File: grafana/.gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 /grafana-6.4.3
 /grafana-6.4.3.tar.gz
 /grafana_webpack-6.4.3.f5f5731149f78e9d7039e56d7963f03bca74f948e40db979b59bbb2f1437dc47.tar.gz
+/grafana-6.4.3
```
Comment (jwendell) [inline grafana/.gitignore:4]: nit: these are duplicate

**Review by jwendell — APPROVED** [review]

File: grafana/.gitignore (hunk 1 above)
Comment (brian-avery) [reply grafana/.gitignore:4]: they are. I could have sworn we fixed this so we didn't have to append new lines every time.

Comment (knrc) [reply grafana/.gitignore:4]: The update.sh script just blindly appends them

Comment (knrc) [reply grafana/.gitignore:4]: Note that in one of the other update.sh scripts I added the following
```
function gitignore() {
  local ignored=$1
  if [ -z "$(egrep '^$ignored$' .gitignore)" ] ; then
    echo "$ignored" >> .gitignore
  fi
}
```

Comment (knrc) [reply grafana/.gitignore:4]: actually that code doesn't work properly but it was the right idea

Comment (knrc) [reply grafana/.gitignore:4]: This is what it should be

```
function gitignore() {
  local ignored=$1
  if [ -z "$(egrep '^'$ignored'$' .gitignore)" ] ; then
    echo "$ignored" >> .gitignore
  fi
}
```

Comment (knrc) [reply grafana/.gitignore:4]: and it looks as if this was added to the grafana update.sh but it didn't work because of the missing 's

Comment (brian-avery) [reply grafana/.gitignore:4]: Should just be able to use a wildcard

**Review by brian-avery — APPROVED** [review]

**Pull request merged**